lib/garage/controller_helper.rb

```diff
@@ -34,7 +34,7 @@ def resource_owner_id
     # Use this method to render 'unauthorized'.
     # Garage user may overwrite this method to response custom unauthorized response.
     # @return [Hash]
-    def unauthorized_render_options
+    def unauthorized_render_options(error: nil)
```

Comment:
We don't need `error` here, can we just write `unauthorized_render_options(*)` to keep compatibility for doorkeeper 2.x?